fares.md

```diff
@@ -23,7 +23,7 @@ on `fare_class`, and the cost of the fare is returned by the price field.
 
 *[`routes_ft.txt`](/files/routes_ft.md)*
 
-`route_id`	| `mode` 		| `fare_class` 	| `proof_of_payment`										
+`route_id`	| `mode` 		| `fare_period` 	| `proof_of_payment`										
```

Comment:
I thought the routes_ft.txt file no longer has a fare_period associated since the fare_id is associated in fare_rules.txt (and the fare_period associated by time definition and the fare_id).